From 2c9f3faac7f545a8415c63f4e8a53e97cdaf6c32 Mon Sep 17 00:00:00 2001
From: Sean Mackrory <mackrorysd@apache.org>
Date: Mon, 5 Jun 2017 15:35:41 -0600
Subject: [PATCH 2602/2848] HADOOP-14433. ITestS3GuardConcurrentOps should not
 require a configured table name.

(cherry picked from commit 76b0751fe43e0e645492497d89ca0ec0b6758461)

Change-Id: If8755130a440f49f1dc4caf721f73284d63ed3b6
---
 .../fs/s3a/s3guard/ITestS3GuardConcurrentOps.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardConcurrentOps.java b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardConcurrentOps.java
index 6b22510..a6718f6 100644
--- a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardConcurrentOps.java
+++ b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardConcurrentOps.java
@@ -78,7 +78,7 @@ public void testConcurrentTableCreations() throws Exception {
             Constants.S3GUARD_METASTORE_DYNAMO));
 
     DynamoDBMetadataStore ms = new DynamoDBMetadataStore();
-    ms.initialize(conf);
+    ms.initialize(getFileSystem());
     DynamoDB db = ms.getDynamoDB();
 
     String tableName = "testConcurrentTableCreations" + new Random().nextInt();
-- 
1.7.9.5

